### ISSUE TITLE
Establish data format and fields

### ISSUE THREAD

**Issue by TimVanMourik** [TimVanMourik/NeuroimagingThroughTheYears#3]
If we want to make a data-driven visualisation over time of neuroimaging methods, we need a database with the relevant fields. These can be subdivided into required and optional fields. This is an issue for listing and naming them.

`class Publication`:
Required:
* `date`: date of publication.
  * Probably only the year is important, but it doesn't hurt recording the full date. If only the year is now, the default month/day will be Jan 1st.
on, but let's keep it simple for now.
* `spatial_resolution`: units: millimetres. 
  * Either two values (min and max) or explicitly separated in two variables. Meaning discussed in #2
 * `temporal_resolution`: units: seconds. 
  * Either two values (min and max) or explicitly separated in two variables. Meaning discussed in #2
* `title`: self-explanatory

Optional:
* `doi`: digital object identifier
  * link to publication. 
* `authors`: list of strings. 
  * It can be considered making this an object that links to ResearchGate and GoogleScholar profiles later 

Considerable:
* `added_by`: 
  * If later on we want to allow people to add content, this might be a useful field.

NOTE: The issue brief will be updated based on the suggestions in this thread.

**Comment by Remi-Gau** [comment]
I agree that need those basics at least. 

Tempted to add `species` because the results might be an interesting dimension to use as a filter.

Was also thinking that for some modalities we could add a couple of extra-fields: e.g brain coverage and field strength might make sense for MRI.

**Comment by TimVanMourik** [comment]
Uh ooh, modality-specific is interesting, but overhead. I'm inclined to have this as a long-term project, but short-term decisions affect how easy this is going to be down the line. Specifically, some data structures and databases (issue #3) work better than others (e.g., JSON-schemes instead of tables, Postgres/MongoDBinstead of MySQL).

My suggestion is: let's keep track of modality-specific variables in this issue. Additionally, plan for a json database and convert the (c/t)sv-files to that.

**Comment by Remi-Gau** [comment]
Yup. SGTM so let's keep the fields described in #2 (i.e one temporal and one spatial resolution) as bare minimum for every new entry, other modality specific fields can be added but are not obligatory.

You know way more than I do about correct format for databases so I will defer to you there.
